src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -203,8 +204,8 @@ public Token call() throws SaltStackException {
      * @return object representing the scheduled job
      * @throws SaltStackException if anything goes wrong
      */
```

Comment:
It was wrong all the time. It seems that the comment itself is 1:1 copied from Salt API doc, which is wrong.